duo_client/admin.py

```diff
@@ -2640,6 +2680,40 @@ def get_u2ftokens_iterator(self):
         """
         return self.json_paging_api_call('GET', '/admin/v1/u2ftokens', {})
 
+    def get_webauthncredentials(self, limit=None, offset=0):
+        """
+        Retrieves a list of webauthn credentials
+        Args:
+            limit: The max number of webauthn credentials to fetch at once. Default None
+            offset: If a limit is passed, the offset to start retrieval.
+                    Default 0
+
+        Returns: A list of webauthn credentials
+
+        Notes: Raises RuntimeError on error.
+        """
+        (limit, offset) = self.normalize_paging_args(limit, offset)
+
+        if limit:
+            return self.json_api_call('GET',
+                                      '/admin/v1/webauthn-credentials',
```

Comment:
This should also call `/admin/v1/webauthncredentials`. We changed this recently.